README.md

```diff
@@ -27,25 +27,54 @@ node node_modules/stryker/dist/src/Stryker.js --help
 ```
 
 ## Configuration
+Options can be configured either via the command line or via a config file.
```

Comment:
This is still a bit unclear. The documentation below suggests that `files` and `mutate` are required, while they are required either using the *old* options or using a config file.